### ISSUE TITLE
[Bug] `DistantClientVersion` returns an error

### ISSUE THREAD

**Issue by pidgeon777** [chipsenkbeil/distant.nvim#90]
`DistantClientVersion` output:

```log
E5108: Error executing lua ...e\pack\packer\start\distant.nvim/lua\distant\command.lua:424: attempt to call method 'new' (a nil value)
stack traceback:
	...e\pack\packer\start\distant.nvim/lua\distant\command.lua:424: in function 'client_version'
	[string ":lua"]:1: in main chunk
```

Neovim version:

```log
NVIM v0.8.3
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilato da runneradmin@fv-az171-224
```

`distant.nvim` setup:

```lua
{
  -- Applies Chip's personal settings to every machine you connect to
  --
  -- 1. Ensures that distant servers terminate with no connections
  -- 2. Provides navigation bindings for remote directories
  -- 3. Provides keybinding to jump into a remote file's parent directory
  ['*'] = require('distant.settings').chip_default(),
  ['133.25.214.41'] = {
    distant = {
      bin = '/root/.cargo/bin/distant',
    },
    lsp = {
      ['Project Distant'] = {
        cmd = { 'clangd-16' },
        filetypes = { 'c', 'cpp' },
        root_dir = '/home/root/projects/src',
        on_attach = function()
          vim.keymap.set('n', 'gD', '<CMD>lua vim.lsp.buf.declaration()<CR>')
          vim.keymap.set('n', 'gd', '<CMD>lua vim.lsp.buf.definition()<CR>')
          vim.keymap.set('n', 'gh', '<CMD>lua vim.lsp.buf.hover()<CR>')
          vim.keymap.set('n', 'gi', '<CMD>lua vim.lsp.buf.implementation()<CR>')
          vim.keymap.set('n', 'gr', '<CMD>lua vim.lsp.buf.references()<CR>')
        end,
      },
    },
  },
}
```

`distant.nvim` branch: v0.2

`distant.exe` path: `C:\Work\Temp\Neovim\data\nvim-data\distant.nvim\bin\distant.exe`

`distant.exe` version: `distant 0.20.0-alpha.4`

Os: Windows 10 Pro

**Comment by pidgeon777** [comment]
Content of `distant.log`:

```log
[ERROR 06/04/2023 17:16:52] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "INVALID_SERVER_JSON"
[ERROR 06/04/2023 17:18:17] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NO_RESULT_CALLBACK_FOUND"
[ERROR 06/04/2023 17:18:24] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "READ_ERROR"
[ERROR 06/04/2023 17:18:27] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NOTIFICATION_HANDLER_ERROR"
[ERROR 06/04/2023 17:18:29] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_REQUEST_HANDLER_ERROR"
[ERROR 06/04/2023 17:18:31] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_RESULT_CALLBACK_ERROR"
[ERROR 06/04/2023 17:18:33] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "ON_INIT_CALLBACK_ERROR"
[ERROR 06/04/2023 17:35:40] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "INVALID_SERVER_MESSAGE"
[ERROR 06/04/2023 17:35:50] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "INVALID_SERVER_JSON"
[ERROR 06/04/2023 17:35:52] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NO_RESULT_CALLBACK_FOUND"
[ERROR 06/04/2023 17:35:53] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "READ_ERROR"
[ERROR 06/04/2023 17:35:55] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NOTIFICATION_HANDLER_ERROR"
[ERROR 06/04/2023 17:35:56] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_REQUEST_HANDLER_ERROR"
[ERROR 06/04/2023 17:35:57] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_RESULT_CALLBACK_ERROR"
[ERROR 06/04/2023 17:36:13] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "ON_INIT_CALLBACK_ERROR"
[ERROR 07/04/2023 14:55:26] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "INVALID_SERVER_JSON"
[ERROR 07/04/2023 14:56:24] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NO_RESULT_CALLBACK_FOUND"
[ERROR 07/04/2023 14:56:29] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "READ_ERROR"
[ERROR 07/04/2023 14:56:31] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NOTIFICATION_HANDLER_ERROR"
[ERROR 07/04/2023 14:56:32] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_REQUEST_HANDLER_ERROR"
[ERROR 07/04/2023 15:01:05] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_RESULT_CALLBACK_ERROR"
[ERROR 07/04/2023 15:01:29] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "ON_INIT_CALLBACK_ERROR"
[ERROR 08/04/2023 03:34:39] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "INVALID_SERVER_JSON"
[ERROR 08/04/2023 03:34:50] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NO_RESULT_CALLBACK_FOUND"
[ERROR 08/04/2023 03:34:52] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "READ_ERROR"
[ERROR 08/04/2023 03:34:54] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "NOTIFICATION_HANDLER_ERROR"
[ERROR 08/04/2023 03:34:56] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_REQUEST_HANDLER_ERROR"
[ERROR 08/04/2023 03:34:57] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "SERVER_RESULT_CALLBACK_ERROR"
[ERROR 08/04/2023 03:34:59] ...packer\start\distant.nvim/lua\distant\cli\client\lsp.lua:151: Client terminated: "ON_INIT_CALLBACK_ERROR"
```

**Comment by chipsenkbeil** [comment]
This is definitely a bug not caught in a refactoring. Thanks for reporting this! I'll get it fixed soon.

**Comment by chipsenkbeil** [comment]
Resolved in the upcoming v0.3 branch.
